string data
---
int64 count
string[] hitIds
string[] workerIds
string[] assignmentIds
